config/locales/en.yml

```diff
@@ -68,7 +68,7 @@ en:
       spree/product:
         name: "Product Name"
         price: "Price"
-        primary_taxon: "Product Category"
+        primary_taxon_id: "Product Category"
```

Comment:
@rioug
It is correct that primary_taxon is now linked to the variant.

However, we still use it at least to collect form input and display the correct error message.
Removing this will result in a validation message **Primary taxon must exist** instead of **Product Category must exist**

Not so sure if users understand that the Primary taxon refers to the Product category in this case.

Reply:
I see, didn't realised that. Sorry could you revert it back to how you did it the first time ? I don't expect user to understand what the Primary taxon refers to.